bindings/python/CouchbaseLite/Collections.py

```diff
@@ -14,7 +14,7 @@
 
 
 # Most general function, accepts params of type FLValue, FLDict or FLArray.
-def decodeFleece(f, *, depth =1, mutable =False):
+def decodeFleece(f, *, depth =99, mutable =False):
```

Comment:
The reason for `depth=1` is to be lazy about creating Python objects, since this can be pretty expensive for a large document.

I think the change to `99` is to work around bugs with document properties that are only partly converted to Python? We should fix those instead. I remember a conversation about this; was it on the CB forums? I just looked for a more recent fix for that, but didn't find one.